conf/docker-aio/c7.dockerfile

```diff
@@ -52,8 +51,8 @@ EXPOSE 9009
 
 RUN mkdir /opt/dv
 
-# yeah - still not happy if glassfish isn't in /usr/local :<
-RUN ln -s /opt/glassfish4 /usr/local/glassfish4
+# keeping the symlink on the off chance that something else is still assuming /usr/local/glassfish4
+RUN ln -s /opt/payara5 /usr/local/glassfish4
```

Comment:
I assume it's safe to not create this symlink. But it doesn't hurt to leave it in. I'm glad the comment was updated to described what's going on.